tox.ini

```diff
@@ -54,6 +54,7 @@ allowlist_externals = /usr/bin/env
 description = Run unit tests
 commands =
     coverage run --source={[vars]src_path} -m pytest {[vars]unit_test_path} -v --tb native -s {posargs}
+    coverage xml
```

Comment:
Why add this?

Reply:
Sorry - was using that for coverage gutters in vscode so I could see what was missed.  Can remove